.github/configs/random1on1-mathserver-config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "guild_id": 894514363350786068
```

Comment:
i mean it can but it doesnt really matter -- anyone can get it by right clicking on the server if they have developer tools enabled. 

they need actual token / admin permissions on the server to actually add bots/do anything.